core/replica.go

```diff
@@ -72,8 +76,14 @@ func New(id uint32, configer api.Configer, stack Stack, opts ...Option) (api.Rep
 		unicastLogs[peerID] = messagelog.New()
 	}
 
-	handleOwnMessage, handlePeerMessage, handleClientMessage := defaultMessageHandlers(id, messageLog, unicastLogs, configer, stack, logger)
-	handleHelloMessage := makeHelloHandler(id, n, messageLog, unicastLogs)
+	var handleOwnMessage, handlePeerMessage, handleClientMessage, handleHelloMessage MessageHandler
+	if nonDefaultStack, ok := stack.(NonDefaultMessageHandler); ok {
+		logger.Info("running with non-default message handlers for system testing\n")
```

Comment:
It seems that we use capitalized style in the existing code consistently (but it does not follow the Go's convention: #105).

```suggestion
		logger.Info("Running with non-default message handlers for system testing")
```